docs/platform/howto/add-storage-space.rst

```diff
@@ -1,7 +1,7 @@
 Add or remove storage 
 =======================
 
-With :doc:`dynamic disk sizing <../concepts/dynamic-disk-sizing>`, you can add or remove disk storage both when you create a service and later for a running service. 
+With :doc:`dynamic disk sizing <../concepts/dynamic-disk-sizing>`, you can add or remove disk storage (by factor of 10 GiB only) both when you create a service and later for a running service. But, it is not possible to add or remove storage when the service nodes are rebalancing.(i.e, during maintenance update, service upgrade, etc.)
 
 .. note::
```

Comment:
1. Below this note, please add a level-1 heading "Use Aiven Console" (so that we have two top level sections in this article: Use Aiven Console, Use Aiven CLI).
2. Also, please change all the headings inside the "Use Aiven Console" section to level-2 headings ("Add storage during new service creation", "Add storage to a running service", "Remove additional storage").
3. Similarly, please change all the headings inside the "Remove additional storage" section to level-3 headings ("Before you start", "Remove added storage").